openapi.yaml

```
@@ -3062,6 +3062,881 @@ paths:
+      parameters:
+      - $ref: '#/components/parameters/pageOffset'
+      - $ref: '#/components/parameters/pageSize'
+      description: |
+        Display all Managed Database node types. The type determines the resources and price of a Managed Database instance.
+
+        Each Managed Database can have one node type. In the case of a high availabilty Database, all nodes are provisioned according to the chosen type.
+
+        **Beta**: This endpoint is in **closed** beta. To get early access and test new Linode products before they
+        hit the market, join the [Linode Green Light](https://www.linode.com/green-light/) program.
+        When accessing this endpoint, please make sure to prepend all requests with `/v4beta` instead of `/v4`, and be
+        aware that this endpoint may receive breaking updates in the future. This notice will be removed when this out of
+        beta.
+      responses:
+        '200':
+          description: Returns a paginated list of all Managed Database types.
+          content:
+            application/json:
+              x-linode-cli-nested-list: cluster_size
+              x-linode-cli-use-schema:
+                type: object
+                properties:
+                  id:
+                    x-linode-cli-display: 1
+                  label:
+                    x-linode-cli-display: 2
+                  cluster_size.quantity:
+                    x-linode-cli-display: 3
+                  cluster_size.price.hourly:
+                    x-linode-cli-display: 4
+                  cluster_size.price.monthly:
+                    x-linode-cli-display: 5
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/PaginationEnvelope'
+                - type: object
+                  properties:
+                    data:
+                      type: array
+                      items:
+                        $ref: '#/components/schemas/DatabaseType'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl https://api.linode.com/v4beta/databases/types
+      - lang: CLI
+        source: >
+          linode-cli databases types
+  /databases/types/{typeId}:
+    parameters:
+      - name: typeId
+        in: path
+        description: The ID of the Managed Database type.
+        required: true
+        schema:
+          type: string
+    x-linode-cli-command: databases
+    get:
+      tags:
+      - Databases
+      summary: Managed Database Type View
+      operationId: getDatabasesType
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: type-view
+      parameters:
+      - $ref: '#/components/parameters/pageOffset'
+      - $ref: '#/components/parameters/pageSize'
+      description: |
+        Display the details of a single Managed Database type. The type determines the resources and price of a Managed Database instance.
```

```suggestion
        Display the details of a single Managed Database type. The type and number of nodes determine the resources and price of a Managed Database instance.
```
Is this better here?